airbyte-server/src/main/java/io/airbyte/server/handlers/MigrationHandler.java

```diff
@@ -70,12 +74,13 @@ public File exportData() {
     try {
       final Path tempFolder = Files.createTempDirectory("airbyte_archive");
       final File archive = Files.createTempFile(ARCHIVE_FILE_NAME, ".tar.gz").toFile();
-      archive.deleteOnExit();
+      fileTtlManager.registerTtl(archive.toPath());
       try {
         exportVersionFile(tempFolder);
         exportAirbyteConfig(tempFolder);
         exportAirbyteDatabase(tempFolder);
         ArchiveHelper.createArchive(tempFolder, archive.toPath());
+        final ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
```

Comment:
What is this for?

Reply:
oh. not needed. was playing with something and forgot to delete.